### DISCUSSION TITLE
Add a chat feature to recovery chat history to new login players.

### DISCUSSION THREAD

**Discussion by sakurawald** [Zrips/CMI#8088] [Feature Requests]
When a player join the server, he can't see any chat messages before.
Is there possible way to let the player see some history chat messages that sent before the player join the server?
Or not only the "chat message", also the other "broadcast messages".

This is just an idea, please have talks and take a consideration.

**Comment by CloudeLecaw** [comment]
a nice idea, but i think thats simply not possible

**Reply by T14D3** [reply]
Very much is possible, can store X amount of last messages sent and send those to players on login

**Reply by CloudeLecaw** [reply]
but these need to be stores somewhere for every player or a an algorithm that knows from the server logs about the last X pms and public chat messages that will be resend on login.

But lets see.
technically still a nice idea.
